frontend/src/components/TilePicker/TilePicker.tsx

```diff
@@ -22,7 +23,14 @@ export function TilePicker<T>() {
           <li
             className={'Tile' + (tile === this.props.value ? ' -Active' : '')}
             key={typeof tile.name === 'string' ? tile.name : tile.name.toString()}
-            onClick={() => this.props.onChange(tile)}
+            onClick={() => {
+              ReactGA.event({
+                category: 'Dataset',
+                action: 'Selected a dataset',
+                label: tile.name.toString()
+              })
+              this.props.onChange(tile)
```

Comment:
Maybe tile.name?

Reply:
That won't compile.

Reply:
tile!.name?

Reply:
Feel free to check and see what compiles